pkg/tests/handlers/accounts.go

```diff
@@ -226,12 +227,26 @@ func TestClusterAuthorizations(ctx context.Context, options *types.TestOptions)
 	return nil
 }
 
+func randomizeResourceName() string {
```

Comment:
Can you add a test for this function, to check that the result from this is included in `helpers.AWSresources`